man/org.freedesktop.portable1.xml

```diff
@@ -193,7 +193,15 @@ node /org/freedesktop/portable1 {
       This method is a superset of <function>GetImageMetadata()</function> with the addition of
       a list of extensions as input parameter, which were overlaid on top of the main
       image via <function>AttachImageWithExtensions()</function>.
-      The <varname>flag</varname> parameter is currently unused and reserved for future purposes.</para>
+      The <varname>flag</varname> parameter can be used to request that, before the units, the path of
+      each extension and an array of bytes with the content of the respective extension-release file
+      are sent. One such structure will be sent for each extension named in the input arguments. The
+      flag value to enable this functionality is defined as follows:</para>
```

Comment:
I am not a fan of this I must say. THis is paramete roverloading, no?

it sounds to me as if this should be a new function that has another array with the extensions with their names and then a nested array of units and fields for the extension release data.

i.e. please let's keep the method call parameter list clean and normalized, i.e. don't pass data through return parameters that actually exist for a different kind of thing.